locales/tr_TR/LC_MESSAGES/duckduckgo.po

```diff
@@ -1563,11 +1563,11 @@ msgstr "İşte beğeneceğinizi düşündüğümüz bazı şeyler"
 #. shown after the DuckDuckGo extension is installed
 msgctxt "welcome message"
 msgid "Here's what it does:"
-msgstr ""
+msgstr "İşte ne var:"
 
 msgctxt "new tab page"
 msgid "Hide Most Visited Sites"
-msgstr ""
+msgstr "En Çok Ziyaret Edilen Siteleri Gizle ""
```

Comment:
Tests are failing because of the double quote at the end of this sting

Reply:
I made the necessary corrections.

Reply:
Not seeing the change. Do you need to `git push`?

Thanks!